.coderabbit.yaml

```
@@ -3,28 +1,33 @@
-    - src/content/*          # Content scripts
-    - src/popup/*            # Popup UI components
-    - src/options/*          # Options page
-    # - src/utils/*            # Shared utilities
-    # - src/*                    # js files
-    # - webpack.config.js      # Build configuration
-
-  ignore_paths:
-    # - dist/                  # Built files
-    - node_modules/
-    - .github/               # CI/CD configurations
+    - src/manifest.json # Core extension file
```

YAMLlint (1.35.1)

[warning] too few spaces before comment

(comments)